packages/expo-image-manipulator/android/build.gradle

```diff
@@ -66,5 +66,6 @@ dependencies {
   unimodule 'unimodules-core'
   unimodule 'unimodules-image-loader-interface'
 
-  api 'com.android.support:support-annotations:27.1.1'
```

Comment:
Compile, target and major of support should match. Se we should wither bump sdk to 28, or leave support annotations as they are.